ui/development.py

```diff
@@ -134,6 +136,30 @@ def throw_404(self, n):
         webbrowser.open(path2)
 
 
+class SvViewSourceForNode(bpy.types.Operator):
+
+    bl_idname = "node.sv_view_node_source"
+    bl_label = "display the source in your editor"
+    kind = StringProperty(default='external')
+    
+    def execute(self, context):
+        n = context.active_node
+        fpath = self.get_filepath_from_node(n)
+
+        with sv_preferences() as prefs:
+            app_name = prefs.external_editor
+            subprocess.call([app_name, fpath])
```

Comment:
Traceback (most recent call last):
  File "/home/ololo/.config/blender/2.78/scripts/addons/sverchok/ui/development.py", line 151, in execute
    subprocess.call([app_name, fpath])
  File "/home/ololo/Downloads/INSTALL/blender-2.78c-linux-glibc219-x86_64/2.78/python/lib/python3.5/subprocess.py", line 557, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/home/ololo/Downloads/INSTALL/blender-2.78c-linux-glibc219-x86_64/2.78/python/lib/python3.5/subprocess.py", line 947, in __init__
    restore_signals, start_new_session)
  File "/home/ololo/Downloads/INSTALL/blender-2.78c-linux-glibc219-x86_64/2.78/python/lib/python3.5/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg)
PermissionError: [Errno 13] Permission denied

Reply:
ok, i'll make a *nix flow control. you'll have to give an example of a working way to start your texteditor with a filename
```text
nikitron$ open some/file/path.py
nikitron$ .\open some/file/path.py
```
?

Reply:
maybe `subprocess.call([app_name, fpath])` doesn't work on your setup...   but something will :)

Reply:
just gedit. but by default we can handle exception with pop up info with howto text

Reply:
just gedit. but by default we can handle exception with pop up info with howto text

Reply:
so you can run 
```python
subprocess.call(["gedit", <some_file_path>])  
```
?

Reply:
can